apps/supernova/src/App.tsx

```diff
@@ -80,7 +79,7 @@ const AppWithNewUrlStructure = (props: any) => {
 
 const StyledApp = (props: any) => {
   return (
-    // @ts-ignore
+    // @ts-ignore > need to validate if theme is part of theme-types
```

Comment:
1) Please remove this line
2) Please run typececk
3) copy error number and message
4) paste instead with @ts-expect-error